presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java

```diff
@@ -202,7 +203,11 @@ public static NullableValue getColumnValue(Row row, int i, CassandraType cassand
                     return NullableValue.of(nativeType, utf8Slice(row.getVarint(i).toString()));
                 case BLOB:
                 case CUSTOM:
-                    return NullableValue.of(nativeType, wrappedBuffer(row.getBytesUnsafe(i)));
+                    ByteBuffer value = row.getBytesUnsafe(i);
```

Comment:
This needlessly checks for array - we only care about the size of the buffer. Whether it's array backed or not is irrelevant.